install.json

```diff
@@ -72,14 +72,30 @@
         },
         "default": "dark",
         "title": "Text shadow color",
-        "order": 4,
+        "order": 3,
         "type": "string"
       },
 
+      "redirect": {
```

Comment:
Instead of a boolean, have a `behavior` object like Tweet This or Particles.

>Behavior
>_When the user clicks the Hero Image..._
>* Scroll down
>* Redirect to URL